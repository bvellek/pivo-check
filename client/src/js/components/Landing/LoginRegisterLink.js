import React from 'react';
import { Link } from 'react-router-dom';

const LoginRegisterLink = (props) => (
  <Link className={props.optionalClass} to="/login">Login/Register</Link>
);

export default LoginRegisterLink;
