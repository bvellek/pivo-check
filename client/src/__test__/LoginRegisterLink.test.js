import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import LoginRegisterLink from '../js/components/Landing/LoginRegisterLink';

describe('<LoginRegisterLink />', () => {
  it('it renders correctly', () => {
    const component = shallow(
      <LoginRegisterLink />
    );
    expect(shallowToJson(component)).toMatchSnapshot();
  });
});
