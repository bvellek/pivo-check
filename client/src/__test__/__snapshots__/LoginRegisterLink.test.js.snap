exports[`<LoginRegisterLink /> it renders correctly 1`] = `
<Link
  replace={false}
  to="/login">
  Login/Register
</Link>
`;
